### PR TITLE
Api (16)

### PR DESCRIPTION
**Pull request by Pawlo77**

### CODE REVIEW

File: containerized_ml_models/api/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+numpy==1.23.4
+pip==22.3.1
+scikit-learn==1.1.3
+scipy==1.9.3
+setuptools==65.5.1
+threadpoolctl==3.1.0
+Werkzeug==2.2.2
+wheel==0.38.4
+xgboost==1.5.1
+zipp==3.10.0
```
Comment (radekaadek) [inline containerized_ml_models/api/requirements.txt:17]: Nice but this file is kinda long and I suggest only adding flask, xgboost and numpy here to make it more readable 📖

File: containerized_ml_models/api/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM continuumio/miniconda3
+
+COPY api.py model.pkl requirements.txt ./
+
+RUN conda create --name virtual_env 
+RUN conda run -n virtual_env pip install -r requirements.txt
```
Comment (radekaadek) [inline containerized_ml_models/api/Dockerfile:6]: You can chain RUN commands with '&& \':
`RUN conda create  --name virtual_env && \`
`conda run -n virtual_env pip install -r requirements.txt`

**Review by radekaadek — APPROVED** [review]
Great solution but try to avoid using pip freeze

**Review by radekaadek — APPROVED** [review]
Nice :D

**Comment by matined** [comment]
Hi, can you change a dictionary name from `api` to `model_api_16`? We'll merge it after :))

**Review by matined — APPROVED** [review]

**Pull request merged**